### PR TITLE
Updated version to 3.7.0rc2

### PR DESCRIPTION
**Pull request by acozzette**

### CODE REVIEW

File: Protobuf.podspec [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # dependent projects use the :git notation to refer to the library.
 Pod::Spec.new do |s|
   s.name     = 'Protobuf'
-  s.version  = '3.7.0'
+  s.version  = '3.7.0-rc1'
```
Comment (TeBoring) [inline Protobuf.podspec]: https://semver.org/
Podspec follows this. So 3.7.0-rc.1?

File: configure.ac [hunk 2]
```diff
@@ -17,7 +17,7 @@ AC_PREREQ(2.59)
 # In the SVN trunk, the version should always be the next anticipated release
 # version with the "-pre" suffix.  (We used to use "-SNAPSHOT" but this pushed
 # the size of one file name in the dist tarfile over the 99-char limit.)
-AC_INIT([Protocol Buffers],[3.7.0],[protobuf@googlegroups.com],[protobuf])
+AC_INIT([Protocol Buffers],[3.7.0-rc1],[protobuf@googlegroups.com],[protobuf])
```
Comment (TeBoring) [inline configure.ac]: https://github.com/protocolbuffers/protobuf/commit/e107e2d68edb550bc7763db2cbbc463ae264dcce#diff-67e997bcfdac55191033d57a16d1408a
3.7.0-rc-1?

File: csharp/Google.Protobuf.Tools.nuspec [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <title>Google Protocol Buffers tools</title>
     <summary>Tools for Protocol Buffers - Google's data interchange format.</summary>
     <description>See project site for more info.</description>
-    <version>3.7.0</version>
+    <version>3.7.0-rc1</version>
```
Comment (TeBoring) [inline csharp/Google.Protobuf.Tools.nuspec]: https://semver.org/
C# seems also follow this: https://docs.microsoft.com/en-us/dotnet/csharp/versioning

File: java/core/pom.xml [hunk 4]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.google.protobuf</groupId>
     <artifactId>protobuf-parent</artifactId>
-    <version>3.7.0</version>
+    <version>3.7.0-rc1</version>
```
Comment (TeBoring) [inline java/core/pom.xml]: 3.7.0-rc-1

File: php/ext/google/protobuf/package.xml [hunk 5]
```diff
@@ -264,8 +264,8 @@ G  A release.
   </release>
   <release>
    <version>
-    <release>3.7.0</release>
-    <api>3.7.0</api>
+    <release>3.7.0-rc1</release>
```
Comment (TeBoring) [inline php/ext/google/protobuf/package.xml]: 3.7.0rc1
https://github.com/protocolbuffers/protobuf/commit/15f4db6cb177fc6fdd3743aae54a8ad30fc55c0b#diff-8620ff0bb4717d44f0449a9697f99e41

File: protoc-artifacts/pom.xml [hunk 6]
```diff
@@ -8,7 +8,7 @@
   </parent>
   <groupId>com.google.protobuf</groupId>
   <artifactId>protoc</artifactId>
-  <version>3.7.0</version>
+  <version>3.7.0-rc1</version>
```
Comment (TeBoring) [inline protoc-artifacts/pom.xml]: 3.7.0-rc-1

File: python/google/protobuf/__init__.py [hunk 7]
```diff
@@ -30,7 +30,7 @@
 
 # Copyright 2007 Google Inc. All Rights Reserved.
 
-__version__ = '3.7.0'
+__version__ = '3.7.0-rc1'
```
Comment (TeBoring) [inline python/google/protobuf/__init__.py]: 3.7.0rc1

File: ruby/google-protobuf.gemspec [hunk 8]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = "google-protobuf"
-  s.version     = "3.7.0"
+  s.version     = "3.7.0.rc1"
```
Comment (TeBoring) [inline ruby/google-protobuf.gemspec]: 3.7.0.rc.1

File: java/core/pom.xml (hunk 4 above)
Comment (acozzette) [reply java/core/pom.xml]: I actually already pushed the artifacts for this one so I think it's too late to change for Java.

File: protoc-artifacts/pom.xml (hunk 6 above)
Comment (acozzette) [reply protoc-artifacts/pom.xml]: I also already pushed the rc1 protoc artifacts so this one is too late to change as well.

**Comment by acozzette** [comment]
I went ahead and updated the version numbers to rc2.

**Pull request merged**